google-cloud-spanner-executor/src/main/java/com/google/cloud/executor/spanner/WorkerProxy.java

```diff
@@ -95,10 +98,22 @@ public static void main(String[] args) throws Exception {
     usePlainTextChannel = commandLine.hasOption(OPTION_USE_PLAIN_TEXT_CHANNEL);
     enableGrpcFaultInjector = commandLine.hasOption(OPTION_ENABLE_GRPC_FAULT_INJECTOR);
 
+    if (commandLine.hasOption(OPTION_MULTIPLEXED_SESSION_OPERATIONS_RATIO)) {
+      multiplexedSessionOperationsRatio =
+          Double.parseDouble(
```

Comment:
Nit: If value is not null, we can add a validation that it is between 0 and 100. Similar validation is present for ports.

Reply:
Added a validation for multiplexed session ratio flag value to be between `0.0` and `1.0`